hr_holidays_public/models/calendar_event.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2023 XCG-CONSULTING - Moctar Diallo <moctarjallo@gmail.com>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, models
+
+
+class CalendarEvent(models.Model):
+    _inherit = "calendar.event"
+
+    @api.model
+    def default_get(self, fields_list):
+        defaults = super().default_get(fields_list)
```

Comment:
Please don't do it this way, as this is affecting all the calendar events. Just pass `"partner_ids": False` in the dictionary creation instead.